erpnext_com/www/docs/user/manual/en/selling/blanket-order.md

```diff
@@ -0,0 +1,33 @@
+<!-- add-breadcrumbs -->
+# Blanker Order
+
+**A Blanket Order is an order from a customer to supply material for specific items over a period of time at pre-negotiated rates.**
+
+From the blanker order, sales orders can be generated as per the delivery schedule provided by the customer. Blanket orders help customers to avoid storing material in huge quantity while allowing to take advantage of discounted rates by committing volumes within a specific time period.
+
+### 1. How to Create a Blanket Order
+
+To create a new Blanket Order:
+
+1. Go to **Selling > Sales > Blanket Order > New**.
```

Comment:
this navigation is not valid, any PR sent?

Reply:
https://github.com/frappe/erpnext/pull/17582